README.md

```diff
@@ -85,6 +85,7 @@ fluent_logger.post('slack', {
 |auto_channels_create|Create channels if not exist. Not available for Incoming Webhook mode (since Incoming Webhook is specific to a channel). A web api `token` for Normal User is required (Bot User can not create channels. See https://api.slack.com/bot-users)|false|
 |https_proxy|https proxy url such as `https://proxy.foo.bar:443`|nil|
 |verbose_fallback|Originally, only `title` is used for the fallback which is the message shown on popup if `title` is given. If this option is set to be `true`, messages are also included to the fallback attribute|false|
+|encoding|The encoding after conversion of the ASCII-8BIT encoded input string. This uses ruby's String#force_encoding. See https://docs.ruby-lang.org/en/trunk/String.html#method-i-force_encoding. You can get supported encoding list by typing `ruby -e 'p Encoding.name_list.sort'`. NOTE: Only works for Incoming Webhook mode and Web API mode.|UTF-8|
```

Comment:
> Just do scrub if we get Encoding::UndefinedConversionError

If I merely scrub ASCII-8BIT encoded UTF-8 string, the exception was not resolved so I explicitly force_encoding(Encoding::UTF_8).

If the way of confirmation is wrong, please point out.

```diff
❯❯❯ git diff
diff --git a/lib/fluent/plugin/slack_client.rb b/lib/fluent/plugin/slack_client.rb
index 899a443..87a2eb5 100644
--- a/lib/fluent/plugin/slack_client.rb
+++ b/lib/fluent/plugin/slack_client.rb
@@ -125,7 +125,6 @@ module Fluent
         when Array
           params.each {|elm| recursive_transcode(elm)}
         when String
-          params.force_encoding(encoding) if params.encoding == Encoding::ASCII_8BIT
           params.scrub! if params.respond_to?(:scrub!)
         else
           params
```

```
Failure:
  Exception raised:
  Encoding::UndefinedConversionError(<"\xE3" from ASCII-8BIT to UTF-8>)
test_post_message_ascii8bit_encoded_utf8_text(SlackClientTest)
/path/to/fluent-plugin-slack/test/plugin/test_slack_client.rb:285:in `block in test_post_message_ascii8bit_encoded_utf8_text'
     282:     # IncomingWebhook posts "#general インストール�"
     283:     def test_post_message_ascii8bit_encoded_utf8_text
     284:       [@incoming].each do |slack|
  => 285:         assert_nothing_raised do
     286:           slack.post_message(default_payload(slack).merge({
     287:             text: invalid_ascii8bit_encoded_utf8_string,
     288:           }))
/path/to/fluent-plugin-slack/test/plugin/test_slack_client.rb:284:in `each'
/path/to/fluent-plugin-slack/test/plugin/test_slack_client.rb:284:in `test_post_message_ascii8bit_encoded_utf8_text'
```